react/package.json

```diff
@@ -10,6 +10,7 @@
     "js-cookie": "^2.2.0",
     "prop-types": "^15.5.10",
     "ramda": "^0.25.0",
+    "react-device-detect": "^1.17.0",
```

Comment:
Did you think of `vtex.device-detector`?
https://github.com/vtex/admin-orders/blob/master/manifest.json#L14

Reply:
The vtex library does not have `isSafari, isChrome, etc...`